org.mwc.debrief.lite/src/main/java/org/mwc/debrief/lite/DebriefLiteApp.java

```diff
@@ -973,4 +982,19 @@ protected void update(final Layers theData, final Plottable newItem,
   {
     _instance.getLayerManager().updateData((Layer) theLayer, newItem);
   }
+  public static String getLastFileLocation()
+  {
+    return _instance.app.getProperty("last_file_location");
```

Comment:
@ayeshare - this is a valid solution, but I don't like these "magic strings".  I'll replace them with constants.  But, I'll also do some refactoring, to try to avoid these static methods.   

The app has the concept of preferences already.  I'll move knowledge of the String back to the `SaveAs` (or relevant class), and then just use the existing preferences.